_schema.graphql

```diff
@@ -4097,8 +4097,8 @@ input InitialOfferInput {
   # ID of order
   orderId: ID!
 
-  # Offer amount in cents
-  amountCents: Int!
+  # Offer amount
+  offerAmount: MoneyInput
```

Comment:
I fear that we're recreating this potential bug in a different form. We call it `cents` to make that data expectation explicit. We should add `dollars` here to avoid the same ambiguity. (I'm open to other ways of referring to the units and subunits, like "major" and "minor," but "dollars" is the natural one compared to "cents.")

Reply:
Unless... `MoneyInput` already embeds that information?

Reply:
I still think this is called an “offer price”. An example can be found [here](https://support.saatchiart.com/hc/en-us/articles/214385258-How-do-I-submit-an-offer-): “Enter your offer price.”

That gets rid of the awkward double use of the word ‘amount’ and I also think that, because this mutation is about making an offer, you can just call the field `price`.